web/src/features/fba/components/map/FBAMap.tsx

```diff
@@ -105,23 +91,12 @@ const FBAMap = (props: FBAMapProps) => {
     }
   })
   const classes = useStyles()
-  const dispatch: AppDispatch = useDispatch()
   const { stations } = useSelector(selectFireWeatherStations)
-  const { values, loading } = useSelector(selectValueAtCoordinate)
   const [showRawHFI, setShowRawHFI] = useState(false)
```

Comment:
we can remove `showRawHFI` state too, can't we?

Reply:
pretend I added that comment to the line below....

Reply:
Yep, thanks